settings/common.py

```diff
@@ -65,6 +65,8 @@
     'rest_framework',
     'rest_framework_docs',
     'rest_framework_expiring_authtoken',
+    'sass_processor',
```

Comment:
@mayank-agarwal-96 we don't want to use `sass_processor`. Can you please use plain css and add it as static files. The reason why I am saying this is that we don't want to add more dependencies at the backend. Also, since this is just a minor feature, so it is an overkill to use `sass_processor` at the backend.